show.go

```diff
@@ -112,13 +113,13 @@ func (s *Session) GetSeasons(id int) ([]Season, error) {
 		seasons[k].FirstTime, err = time.Parse("02/01/2006 15:04", v.FirstTimeRaw)
 
 		if err != nil {
-			return nil, err
+			log.Print(err)
```

Comment:
Because I wasn't sure what else to do, and I wanted to see it in the logs when data wasn't being loaded